### DISCUSSION TITLE
Kreya into CI/CD

### DISCUSSION THREAD

**Discussion by umang1402** [riok/Kreya#24] [General]
Currently can Kreya be integrated with existing CI/CD pipeline

**Comment by CommonGuy** [comment]
Kreya currently cannot be integrated into a CI/CD pipeline (or it wouldn't be useful). At the moment, Kreya does not have any support for tests or response assertions, which is why there is no "Kreya CLI" yet.

We may add these features to Kreya, since they are really useful for E2E testing of gRPC APIs.

**Reply by umang1402** [reply]
Thanks a lot for the quick response :)

**Comment by CommonGuy** [comment]
The Kreya CLI has been available since some time, see the docs: https://kreya.app/docs/cli/.

So Kreya can now be integrated into a CI/CD pipeline.
